WordPress/openverse
# Fix banner close button hover state

## Fixes



Fixes #2203 by @fcoveram 

## Description


This PR updates the `VCloseButton`'s "black" variant to use transparent background, white icon, and white background with opacity-10 on hover. This button is used in the pages menu modal on the mobile screen and the black notification banner on the yellow home page.

The banner close button's size changed from "tiny" to "small" as in the mockups.

## Testing Instructions

Open the app in an incognito mode to make sure that the privacy banner shows up. Check that the close button is visible, and it changes background color on hover.
Narrow the screen and click on the "menu" button to open the pages menu modal. The close button should look the same as the banner button.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
